src/main.c

```diff
@@ -2247,7 +2251,8 @@ void on_key(GLFWwindow *window, int key, int scancode, int action, int mods) {
         }
     }
     if (!g->typing) {
-        if (key == CRAFT_KEY_FLY) {
+        /**This condition is modified to enable instant High fly mode.*/
```

Comment:
A more straight forward comment instead of this would be, "Allow user key press to initiate "Fly high"".

Reply:
change done.